pyaedt/hfss.py

```diff
@@ -5728,10 +5728,9 @@ def get_antenna_ffd_solution_data(
         sphere_name=None,
         variations=None,
         overwrite=True,
-        taper="flat",
     ):
-        """Export antennas parameters to Far Field Data (FFD) files and return the ``FfdSolutionData`` object. For
-        phased array cases, only one phased array will be calculated.
+        """Export antennas parameters to Far Field Data (FFD) files and return the ``FfdSolutionDataExporter`` object.
+        For phased array cases, only one phased array is calculated.
```

Comment:
```suggestion
        
        For phased array cases, only one phased array is calculated.
```
Blank line needs to separate brief description and additional information.